kvmagent/kvmagent/plugins/ha_plugin.py

```
@@ -14,6 +14,7 @@
 from zstacklib.utils import xmlobject
 from zstacklib.utils import jsonobject
 from zstacklib.utils import iscsi
+from zstacklib.utils import lock
```

Ruff (0.12.2)

`zstacklib.utils.lock` imported but unused

Remove unused import: `zstacklib.utils.lock`

(F401)